.gitignore

```diff
@@ -1,6 +1,9 @@
 # Bundler config
 /.bundle
 
+# local ruby env
+.ruby-version
+
```

Comment:
I added this for me. My machine doesn't like `2.2.2`, so I'm using `2.4.0`. I added this so that git stops complaining that I need to commit my modified file.